### ISSUE TITLE
Some packages can't be discovered

### ISSUE THREAD

**Issue by jkonecny12** [fedora-infra/fedora-packages#432]
Hello everyone,

I wanted to look on [kakoune](https://src.fedoraproject.org/rpms/kakoune) package for versions but I had no luck with fedora packages site. Could you please look on what the problem is?

https://apps.fedoraproject.org/packages/s/kakoune

**Comment by cverna** [comment]
I am trying to run the indexing, it has been failing a lot recently due to threads loosing connections with mdapi.fedoraproject.org.

Let's see if we can get it to work :smile:

**Comment by maflcko** [comment]
Same for podman: https://duckduckgo.com/?q=fedora+podman+package&t=ffab&ia=web

**Comment by richardfearn** [comment]
Also a few more:

* https://apps.fedoraproject.org/packages/s/clide
* https://apps.fedoraproject.org/packages/s/colordiff
* https://apps.fedoraproject.org/packages/s/disktype
* https://apps.fedoraproject.org/packages/s/perl-App-ccdiff
* https://apps.fedoraproject.org/packages/s/php-pdb
* https://apps.fedoraproject.org/packages/s/txt2regex
* https://apps.fedoraproject.org/packages/s/whowatch

**Comment by richardfearn** [comment]
#411 has some useful information - links to mdapi for example.

As a test I tried https://mdapi.fedoraproject.org/rawhide/srcpkg/clide, and it is there.

**Comment by chkr-private** [comment]
Same for scummvm:  https://apps.fedoraproject.org/packages/s/scummvm

**Comment by eclipseo** [comment]
Same for Rclone https://apps.fedoraproject.org/packages/s/rclone

**Comment by jkonecny12** [comment]
Also packit https://apps.fedoraproject.org/packages/s/packit

**Comment by cverna** [comment]
I ll try to run another indexing but I do not guarantee the results will be better that what we have now :S

**Comment by sergiomb2** [comment]
> I ll try to run another indexing but I do not guarantee the results will be better that what we have now :S

I tested and worked , now seems have much less entries again ...

**Comment by ferdnyc** [comment]
I closed my dupe report #436, but the strange redirect I noted there seems significant:
```console
$ curl https://apps.fedoraproject.org/packages/python-colorlog/bugs/all
<html>
 <head>
<script type="text/javascript" src="/packages/tw2/resources/tw2.jquery/static/jquery/1.7.1/jquery.js" ></script>

<script type="text/javascript" src="/packages/tw2/resources/moksha.wsgi.widgets.moksha_js/static/moksha.js" ></script>

<script type="text/javascript" src="/packages/tw2/resources/fedoracommunity.connectors.widgets.widgets/static/js/fcomm.js" ></script>
  <title>302 Found</title>
 </head>
 <body>
  <h1>302 Found</h1>
  The resource was found at <a href="http://localhost:10016/packages/s/python-colorlog">http://localhost:10016/packages/s/python-colorlog</a>;
you should be redirected automatically.


 </body>
</html>
```

...Redirect to `localhost`!? Even if that's just how Moksha expresses "No data found"... it should probably find a _different_ way to say "No data found".

**Comment by rugk** [comment]
The search is totally broken and even when trying to access the links directly you are redirected to the search, i.e. the whole website is broken:
https://apps.fedoraproject.org/packages/lollypop/
https://apps.fedoraproject.org/packages/dino

**Comment by ferdnyc** [comment]
> The search is totally broken and even when trying to access the links directly you are redirected to the search, i.e. the whole website is broken:

Yeah, and it's especially unfortunate because all of the https://bugz.fedoraproject.org/ bug links in RPM metadata are redirected through apps.fedoraproject.org. Which means the bug lists are inaccessible for all of the missing packages.

**Comment by ferdnyc** [comment]
In fact, given the problem described in my previous comment, and since it seems this situation is not  a short-term fix, is there any way requests to https://apps.fedoraproject.org/packages/s/<var>packagename</var> could just be blanket-redirected somewhere else? The simplest solution would be https://src.fedoraproject.org/rpms/<var>packagename</var>, which is better than nothing — and "nothing" is precisely what a lot of packages currently have.

I was just on a GitHub page for a GEdit plugin, and the "Fedora official repo" link went to https://apps.fedoraproject.org/packages/s/gedit-control-your-tabs ... which is currently absent.

There are a _lot_ of links out there pointing to Fedora Packages listings, and it's a shame to leave them all in limbo the way they are right now.

**Comment by jonasmalacofilho** [comment]
One more:

 - https://apps.fedoraproject.org/packages/s/liquidctl

**Comment by richardfearn** [comment]
From earlier comments, it doesn't sound like missing packages can be added individually, so there's probably not much point listing them in this issue :(

**Comment by jkonecny12** [comment]
Even if they can't be added individually it makes a good testing sample to list them. Also shows how big the problem is.
